src/basics.php

```diff
@@ -15,9 +15,9 @@
 use Cake\Core\Configure;
 use Cake\Error\Debugger;
 
-/**
- * Basic defines for timing functions.
- */
+    /**
+     * Basic defines for timing functions.
+     */
     define('SECOND', 1);
     define('MINUTE', 60);
     define('HOUR', 3600);
```

Comment:
Ah yeah, my new sniffer caught these as `T_CONST`s... so I slapped this PR in without using my peripheral vision. Interesting.